code/components/voip-mumble/component.json

```diff
@@ -12,7 +12,8 @@
 		"vendor:webrtc-audio-processing",
 		"vendor:labsound",
 		"vendor:speexdsp",
-		"scrbind:base"
+		"scrbind:base",
+		"gta:core:five"
```

Comment:
Plain `gta:core` or maybe just ICoreGameInit usage instead of OnKillNetworkDone?